src/test/java/uk/gov/pay/adminusers/utils/email/EmailValidatorIsPublicSectorEmailTest.java

```diff
@@ -78,6 +78,7 @@ public EmailValidatorIsPublicSectorEmailTest(String email, boolean testResult) {
                 {"valid@slc.co.uk", true},
                 {"valid@ucds.email", true},
                 {"valid@acas.org.uk", true},
+                {"valid@acas.nhm.ac.uk", true},
```

Comment:
I think here we are testing the "main domains" in the list, for example `valid@nhm.ac.uk`, because we have "subdomains" section below

Reply:
ups... copy pasting exception. Thanks!